### PR TITLE
Update Webpack

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/calculation/convert-currency.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  const currencyArray = valueInPounds.split('.')
   const pounds = currencyArray[0]
   const pence = (currencyArray[1] || '00').padEnd(2, '0')
   return Number(pounds + pence)
 }
 
-function convertToPounds (valueInPounds) {
-  return valueInPounds / 100
+function convertToPounds (valueInPence) {
+  return valueInPence / 100
```
Comment (johnwatson484) [inline app/calculation/convert-currency.js:9]: These variable names were the wrong way around.  No functionality change, just corrected naming so they make sense.

File: app/calculation/index.js [hunk 2]
```diff
@@ -7,10 +7,10 @@ const schemeYears = require('./scheme-years')
...
   const schemeYearValues = !isNaN(value)
     ? schemeYears.map(schemeYear => ({ schemeYear, value }))
-    : [{ schemeYear: 2021, value: value.value2021 || 0 },
-        { schemeYear: 2022, value: value.value2022 || 0 },
-        { schemeYear: 2023, value: value.value2023 || 0 },
-        { schemeYear: 2024, value: value.value2024 || 0 }]
+    : [{ schemeYear: 2021, value: value.value2021 || 0 }, // eslint-disable-line
+      { schemeYear: 2022, value: value.value2022 || 0 }, // eslint-disable-line
+      { schemeYear: 2023, value: value.value2023 || 0 }, // eslint-disable-line
+      { schemeYear: 2024, value: value.value2024 || 0 }] // eslint-disable-line
```
Comment (johnwatson484) [inline app/calculation/index.js:13]: Latest version of StandardJS is struggling to evaluate this line consistently.  Sometimes believe indentation should be 6 spaces and sometimes 8.

Probably a sign that we should simplify this and not have ternary operator.  Have left for now as changing calculation code is outside scope of WebPack update.

File: test/acceptance/features/prcalculator.feature [hunk 3]
```diff
@@ -17,7 +17,7 @@ Then I expect that element "h1" contains the text "Agricultural Transition Plan
 Scenario: 003-User can click on Farming is changing link
 Given I open the url "/"
 When I click on the link "Farming is changing"
-Then I expect that the url contain "/986879/farming-changing.pdf"
+Then I expect that the url contain "/farming-changing.pdf"
```
Comment (johnwatson484) [inline test/acceptance/features/prcalculator.feature:20]: Published URL changes periodically beyond our control.  Tying the file name to a specific Id results in frequent test fails as the Id isn't static.

**Review by steve-dickinson — APPROVED** [review]
LGTM

**Pull request merged**